hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/rollback/BaseRollbackActionExecutor.java

```diff
@@ -123,6 +124,11 @@ private HoodieRollbackMetadata runRollback(HoodieTable<T, I, K, O> table, Hoodie
     // Finally, remove the markers post rollback.
     WriteMarkersFactory.get(config.getMarkersType(), table, instantToRollback.requestedTime())
         .quietDeleteMarkerDir(context, config.getMarkersDeleteParallelism());
+    if (table.getMetaClient().getTableConfig().getTableVersion().lesserThan(HoodieTableVersion.EIGHT)) {
+      // For MOR table rollbacks, rollback command blocks might generate markers under rollback instant. So, lets clean up the markers if any.
+      WriteMarkersFactory.get(config.getMarkersType(), table, rollbackInstant.requestedTime())
+          .quietDeleteMarkerDir(context, config.getMarkersDeleteParallelism());
```

Comment:
This is already done as part of L125 in this class.  What's the reason of adding this?

Reply:
the prev L is for the commit being rolled back
and this L 130 is for the rollback instant. 

for eg, 
t5.dc 
started and crashed. 

t6.rb

L125 is for deleting markers for t5 instant. 
and L 130 is for deleting markers for t6 instant.

looks like he has added a comment in L 128.

Reply:
I see, the instant time is different; that's how I got confused.